modules/core/src/test/scala/io/github/irevive/union/derivation/ComplexTypeDerivationSuite.scala

```diff
@@ -0,0 +1,54 @@
+package io.github.irevive.union.derivation
+
+class GenericTypeDerivationSuite extends munit.FunSuite {
```

Comment:
Should the file also be called `GenericTypeDerivationSuite.scala` rather than `ComplexTypeDerivationSuite.scala`?

Reply:
Sure, my IDE was playing tricks on me :wink:

Reply:
done!